cirq-google/cirq_google/engine/virtual_engine_factory_test.py

```diff
@@ -51,6 +51,23 @@ def test_create_from_device():
     _test_processor(engine.get_processor('sycamore'))
 
 
+def test_median_device():
+    q0, q1 = cirq.GridQubit.rect(1, 2, 5, 3)
+    cal = factory.load_median_device_calibration('rainbow')
+    # Spot-check an arbitrary set of values to confirm we got the right data.
+    assert cal['parallel_p00_error'][(q0,)][0] == 0.0067149999999998045
```

Comment:
Checking equality on floats makes me nervous.  Worth changing to isclose?

Reply:
This is actually intentional - the floats are hardcoded, so using exact equality provides greater certainty that we got the desired data.

Reply:
Are you sure this works?  Often when converting string floats, we can get conversion errors, especially if there are 32/64 bit conversion issues anywhere.  I still really think this is error-prone, since conversion from text can introduce the equivalent of an off-by-one precision error in the float.

Reply:
That's fair. Switched to `isclose`.